packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -133,6 +133,9 @@ export const topNTableSchema = {
     description: {
       type: 'string',
     },
+    entityLink: {
+      type: 'string',
+    },
```

Comment:
This will probably need to be an array of strings

Reply:
just one string. The urls for an entity type have the same format